index.js

```diff
@@ -18,16 +18,16 @@ const buildConfig = (keys, initialConfig, prefix, { removePrefixes }) => {
 };
 
 /**
- * NextJS plugin to pass environment variables to NextJS's runtime configuration (`publicRuntimeConfig` and `serverRuntimeConfig`)
+ * NextJS plugin to pass environment variables to NextJS's configuration (`publicRuntimeConfig`, `serverRuntimeConfig` and `env`)
  *
  * @param {Object} options
  * @param {String} options.publicPrefix     Prefix of variables to lookup and then pass to publicRuntimeConfig
  * @param {String} options.serverPrefix     Prefix of variables to lookup and then pass to serverRuntimeConfig
  * @param {String} options.buildPrefix     Prefix of variables to lookup and then pass to buildRuntimeConfig
```

Comment:
Not related but I think we are missing a space here.

Reply:
We do 😄